modules/nf-core/kraken2/kraken2/meta.yml

```diff
@@ -28,12 +28,12 @@ input:
       type: directory
       description: Kraken2 database
   - save_output_fastqs:
-      type: boolean
+      type: string
```

Comment:
Is this not a boolean?

Reply:
we don't have boolean as a type, see (https://nf-co.re/docs/contributing/modules#documentation point 8). Not sure if that is a nextflow limitation (I would be for it)

Reply:
I see that the PR is now closed, but I wanted to clarify this point anyway. I checked the modules repo and when I searched for `type: boolean` there are 72 occurrences in `meta.yml`s files both in modules and subworkflows. So if this is not allowed we should get rid of them. When you say this is a nextflow limitation what do you mean exactly? Thanks!

Reply:
ya i see those 72 too....i think we will have to change those too but,I am also curious as to why boolean is not supported @mashehu . now that we specify as string, will providing `"true"` be the same as `true` ?

Reply:
was just going off of the guidelines, which I think @jfy133 worked on.

Reply:
But since we don't validate these types anywhere (besides in json-schema for meta.yml) and don't use them anywhere AFAIK (besides rendering them on the website), I am all for adding `boolean` as a type to the guidelines.

Reply:
We had it already in the schema, added it also to the guidelines now: https://github.com/nf-core/nf-co.re/pull/1767/files

Reply:
will tools also need an update? as linting with boolean fails